iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
```

Comment:
In config.yaml, we have a "default" node containing all the other nodes. I think we should a similar convention in both files. That does not necessarily need to have the "default" node, but changing may be hard if we need to be backwards compatible.
The other config has `log_file: "/var/log/host-os/builds.log"` and it uses that same path for the three scripts that we have. I think we should use the "/var/log/host-os" here too and (unrelated to this PR) maybe use different files for different scripts (upgrade_versions, create_release_notes).